knexfile.js

```diff
@@ -0,0 +1,17 @@
+const { parse } = require('pg-connection-string')
+
+module.exports = {
+    development: {
+        client: 'pg',
+        connection: {
+            ...parse(process.env.PG_CONNECTION),
+            ssl: { rejectUnauthorized: false },
+        },
+        migrations: {
+            directory: __dirname + '/db/migrations',
+        },
+        seeds: {
+            directory: __dirname + '/db/seeds',
```

Comment:
Is there supposed to be a file in this folder?
Currently there isn't or does it appear in some other way?